alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/Monitor.kt

```diff
@@ -47,6 +47,7 @@ data class Monitor(
     override val schedule: Schedule,
     override val lastUpdateTime: Instant,
     override val enabledTime: Instant?,
+    val schemaVersion: Int = 0,
```

Comment:
There is many other places that require 0 as default schema version. Lets create a constant with  `NO_SCHEMA_VERSION` to use as default.

Reply:
Sure, will change it.